MAINTENANCE.md

```diff
@@ -42,16 +42,33 @@ The main advantages of this are:
 
 We use Github Actions and TravisCI continuous integration (CI) services to
 build and test the project on Linux, macOS and Windows.
-The configuration file for these services are in
-`.github/workflows/ci_tests.yaml` and `.travis.yml`.
 They rely on the `requirements.txt` file to install required dependencies using
 conda and the `Makefile` to run the tests and checks.
-There is also a `.github/workflows/cache_data.yaml` file which caches some GMT
-remote data files needed for the Github Actions CI on a regular basis.
+
+### Github Actions
+
+There are 3 configuration files located in `.github/workflows`:
+
+1. `ci_tests.yaml` (Style Checks, Tests on Linux/macOS/Windows)
+
+This is ran on every commit on the *master* and Pull Request branches.
+It is also scheduled to run daily on the *master* branch.
```

Comment:
Ok, I've reorganized the Continuous Integration section a bit since it was getting complicated. You can insert a comment here about 'skipping ci-tests' later in #534.